reana-ui/src/pages/workflowList/components/WorkflowActionsPopup.js

```diff
@@ -0,0 +1,77 @@
+/*
+  -*- coding: utf-8 -*-
+
+  This file is part of REANA.
+  Copyright (C) 2020 CERN.
+
+  REANA is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+import PropTypes from "prop-types";
+import React, { useState } from "react";
+import { Icon, Menu, Popup } from "semantic-ui-react";
+import { useDispatch } from "react-redux";
+
+import { deleteWorkflow } from "../../../actions";
+
+export default function WorkflowActionsPopup({
+  workflow,
+  setOpenDeleteModal,
+  setSelectedWorkflow,
+}) {
+  const dispatch = useDispatch();
+  const [open, setOpen] = useState(false);
+  const { id, name, run, size, status } = workflow;
+  let menuItems = [];
+  if (status === "deleted") {
+    if (size !== "0K") {
+      menuItems.push({
+        key: "freeup",
+        content: "Free up disk",
+        icon: "hdd",
+        onClick: (e) => {
+          dispatch(deleteWorkflow(id, true));
+          setOpen(false);
+          e.stopPropagation();
+        },
+      });
+    }
+  } else {
+    menuItems.push({
+      key: "delete",
+      content: "Delete workflow",
+      icon: "trash",
+      onClick: (e) => {
+        setOpenDeleteModal(true);
+        setSelectedWorkflow({ id, name, run, size });
+        setOpen(false);
+        e.stopPropagation();
+      },
+    });
+  }
+  return (
+    <Popup
+      trigger={
+        <Icon
+          name="ellipsis vertical"
```

Comment:
What do you think if we have a `horizontal ellipsis` in the top right corner with the `Menu` opening downside instead of upside?

Reply:
Do you mean something like this?

![image](https://user-images.githubusercontent.com/8863238/95063069-22c92700-06fe-11eb-9c65-ae0a614fa2cf.png)

vs

![image](https://user-images.githubusercontent.com/8863238/95062840-cd8d1580-06fd-11eb-9047-4c0cd8f507db.png)

I agree on the opening downside, looks better IMO 🙏 Not sure about the horizontal dots.. Opinions @tiborsimko @diegodelemos @audrium

Reply:
I'd give my vote for the vertical option :1st_place_medal:

Reply:
The reason for horizontal dots is that the `Menu` almost takes the space within that workflow box only and in the vertical, it is over the second workflow box as well, it will not create any problem but UI wise the horizontal one looks cleaner. Maybe, we can remove the `pointing` from menu since it is not aligning with the `dots` in both horizontal and vertical.

Reply:
For me it shows very detached from the button:
![image](https://user-images.githubusercontent.com/9035606/95082373-3afb6f00-071b-11eb-85cf-d901a3922bec.png)

If we remove the `pointing` I would expect the box to be closer to the button, what do you think?

Reply:
There's an [`offset`](https://react.semantic-ui.com/modules/popup/#usage-offset) prop to address this, but I cannot manage to make it work. If you manage please let me know.